docs/source/starthere/fundamentals.rst

```diff
@@ -0,0 +1,240 @@
+
+
+Specifying training data
+------------------------
+
+NeMo will handle creation of data loaders for you, as long as you put your data into the expected input format. You may also need to train a tokenizer before starting training. Learn more about data formats for :doc:`LLM <../nlp/nemo_megatron/gpt/gpt_training>`, :doc:`Multimodal <../multimodal/mllm/datasets>`, :ref:`Speech AI <section-with-manifest-format-explanation>`, and :doc:`Vision models <../vision/datasets>`.
+
+
+Model checkpoints
+-----------------
+
+Throughout training, model :doc:`checkpoints <../checkpoints/intro>` will be saved inside ``.nemo`` files. These are archive files containing all the necessary components to restore a usable model, e.g.:
+
+* model weights (``.ckpt`` files),
+* model configuration (``.yaml`` files),
+* tokenizer files
+
+The NeMo team also releases pretrained models which you browse on `NGC <https://catalog.ngc.nvidia.com/models?query=nemo&orderBy=weightPopularDESC>`_ and `HuggingFace Hub <https://huggingface.co/models?library=nemo&sort=downloads&search=nvidia>`_.
+
+
+Finetuning
+----------
+
+NeMo allows you to finetune models as well as train them from scratch.
+
+You can do this by initializing a model with random weights, replacing some/all the weights with those of a pretrained model, and then continuing training as normal, potentially with some small changes such as reducing your learning rate or freezing some model parameters.
+
+
+.. _where_next:
+
+Where next?
+-----------
+
+Here are some options:
+
+* dive in to `examples <https://github.com/NVIDIA/NeMo/tree/main/examples>`_ or :doc:`tutorials <./tutorials>`
+* read docs of the domain (e.g. :doc:`LLM <../nlp/nemo_megatron/intro>`, :doc:`Multimodal <../multimodal/mllm/intro>`, :doc:`ASR <../asr/intro>`, :doc:`TTS <../tts/intro>`, :doc:`Vision Models <../vision/intro>`) you want to work with
+* learn more about the inner workings of NeMo:
+
+  * `NeMo Primer <https://github.com/NVIDIA/NeMo/blob/stable/tutorials/00_NeMo_Primer.ipynb>`_ notebook tutorial
+
+    * hands-on intro to NeMo, PyTorch Lightning, and OmegaConf
+    * shows how to use, modify, save, and restore NeMo models
+
+  * `NeMo Models <https://github.com/NVIDIA/NeMo/blob/stable/tutorials/01_NeMo_Models.ipynb>`__ notebook tutorial
+
+    * explains the fundamentals of how NeMo models are created
+
+  * :doc:`NeMo Core <../core/core>` documentation
+
```

Comment:
suggested revision for consistency in the list.

Here are some options:

- **Explore Examples or Tutorials**: Dive into NeMo by exploring our `examples <https://github.com/NVIDIA/NeMo/tree/main/examples>`_ or :doc:`tutorials <./tutorials>`.
- **Domain-Specific Documentation**:
      - For Natural Language Processing (NLP), check out the :doc:`LLM <../nlp/nemo_megatron/intro>` documentation.
      - For Multimodal tasks, refer to the :doc:`Multimodal <../multimodal/mllm/intro>` documentation.
      - If you’re interested in Automatic Speech Recognition (ASR), explore the :doc:`ASR <../asr/intro>` documentation.
      - For Text-to-Speech (TTS), find details in the :doc:`TTS <../tts/intro>` documentation.
      - Lastly, for Vision Models, consult the :doc:`Vision Models <../vision/intro>` documentation.
- **`NeMo Primer <https://github.com/NVIDIA/NeMo/blob/stable/tutorials/00_NeMo_Primer.ipynb>`_**: This tutorial provides a hands-on introduction to NeMo, PyTorch Lightning, and OmegaConf. It covers how to use, modify, save, and restore NeMo models.
- **`NeMo Models <https://github.com/NVIDIA/NeMo/blob/stable/tutorials/01_NeMo_Models.ipynb>`__ notebook **: In this tutorial, you’ll learn the fundamentals of creating NeMo models.
- **NeMo Core Documentation**: Explore the :doc:`NeMo Core <../core/core>` documentation for NeMo, which delves into the inner workings of the framework.

Reply:
Seems that the RST formatting doesn't allow link text to be bold, so the NeMo Primer and NeMo Models links currently don't display properly.
As a workaround, maybe put the link on the word "tutorial"?

Reply:
Oh know. I didn't know that. Thank you for checking. How about just removing the bold formatting from all of the categories then. It's better to be consistent with the formatting. Makes it easier for the user to scan the info.

Reply:
OK, that looks fine I think. Can you think of a way to get rid of the "delves" in the NeMo Core Documentation bullet point please? :)